extensions/amp-story/1.0/amp-story.js

```diff
@@ -312,6 +312,10 @@ export class AmpStory extends AMP.BaseElement {
       this.variableService_.onNavigationStateChange(stateChangeEvent);
       this.analytics_.onNavigationStateChange(stateChangeEvent);
     });
+
+    // Disallow all actions in a (standalone) story.
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
+    actions.setWhitelist([]);
```

Comment:
@newmuis The error message is not very helpful for `amp-story`. For example:

```
[Action] "tap" is not whitelisted ([]).
```

We can later add a second param to set a custom error message if this becomes a common source of confusion.